legacy/messaging/src/main/java/org/jboss/as/messaging/MigrateOperation.java

```diff
@@ -524,7 +533,13 @@ private void migrateBroadcastGroup(ModelNode newAddOp, List<String> warnings) {
     private void removePropertiesWithExpression(ModelNode newAddOp, List<String> warnings, String... properties) {
         for (String property : properties) {
             if (newAddOp.hasDefined(property) && newAddOp.get(property).getType() == EXPRESSION) {
-                newAddOp.remove(property);
+                if (property.equals(JGROUPS_CHANNEL.getName())) {
+                    ModelNode resolvedNode = newAddOp.get(property).resolve();
```

Comment:
-1 The jgroups-channel and jgroups-stack does not allow expressions in the new messaging-activemq. We can not copy them.
We can neither resolve them before copy because there is no guarantee that the resolved value on the DC would be similar to the resolved value on any domain servers.